morf-oracle/src/main/java/org/alfasoftware/morf/jdbc/oracle/OracleDialect.java

```diff
@@ -423,25 +425,21 @@ public Collection<String> dropStatements(View view) {
 
 
   /**
-   * @see org.alfasoftware.morf.jdbc.SqlDialect#viewDeploymentStatementsAsLiteral(org.alfasoftware.morf.metadata.View)
+   * @see org.alfasoftware.morf.jdbc.SqlDialect#getSqlFrom(org.alfasoftware.morf.sql.element.ClobFieldLiteral)
    */
   @Override
-  public AliasedField viewDeploymentStatementsAsLiteral(View view) {
-    final String script = viewDeploymentStatementsAsScript(view);
-
+  protected String getSqlFrom(ClobFieldLiteral field) {
     final int chunkSize = 512;
-    if (script.length() <= chunkSize) {
-      return super.viewDeploymentStatementsAsLiteral(view);
+    if (field.getValue().length() <= chunkSize) {
+      return getSqlFrom((FieldLiteral) field);
     }
-
-    Iterable<String> scriptChunks = Splitter.fixedLength(chunkSize).split(script);
+    Iterable<String> scriptChunks = Splitter.fixedLength(chunkSize).split(field.getValue());
     FluentIterable<Cast> concatLiterals = FluentIterable.from(scriptChunks)
-      .transform(chunk -> SqlUtils.cast(SqlUtils.literal(chunk)).asType(DataType.CLOB));
+            .transform(chunk -> SqlUtils.cast(SqlUtils.literal(chunk)).asType(DataType.CLOB));
 
-    return SqlUtils.concat(concatLiterals);
+    return String.valueOf(SqlUtils.concat(concatLiterals));

```

Comment:
Are you sure about `String.valueOf()` here?
This should be something like `getSqlFrom(SqlUtils.concat(concatLiterals))`.